include/ace/managers/viewport/tilebuffer.h

```diff
@@ -170,11 +170,6 @@ void tileBufferDestroy(tTileBufferManager *pManager);
  */
 void tileBufferProcess(tTileBufferManager *pManager);
 
-void tileBufferReset(
-	tTileBufferManager *pManager, UWORD uwTileX, UWORD uwTileY,
-	UBYTE ubBitmapFlags, UBYTE isDblBuf
-);
-
```

Comment:
why removed tileBufferReset fn?

Reply:
you're right, I moved it to `tilebuffer.c` for no good reason.

Reply:
moved back to header